mangum/adapter.py

```diff
@@ -85,36 +92,45 @@ def handler(self, event: dict, context: dict) -> dict:
         return response
 
     def handle_http(self, event: dict, context: dict) -> dict:
-        server, client = get_server_and_client(event)
+        is_http_api = 'multiValueQueryStringParameters' not in event
```

Comment:
What about `is_http_api = 'http' not in event["requestContext"]` instead?

Reply:
good idea, done